.changes/v3.6.0/816-improvements.md

```diff
@@ -0,0 +1,4 @@
+* `resource/vcd_network_isolated_v2` add support for `metadata` so this field can be set when creating/updating isolated networks. [GH-816]
+* `datasource/vcd_network_isolated_v2` add support for `metadata` so this field can be retrieved when reading from isolated networks. [GH-816]
```

Comment:
I'm just thinking about the casual readers of the changelog. Maybe we should add "NSX-T" in all these entries, so people know the networking type unambiguously? Something like:

```suggestion
* `datasource/vcd_network_isolated_v2` add support for `metadata` so this field can be retrieved when reading from isolated NSX-T networks. [GH-816]
```

Reply:
Done. Not sure if we should put instead _"NSX-T and NSX-V"_, as if I'm not wrong the **v2** resource supports both. But probably just noting NSX-T is clearer for the user.

Reply:
Ah! @Didainius what do you think?